src/NECompletion/NECPreferences.class.st

```diff
@@ -23,12 +23,12 @@ Class {
 		'smartCharactersWithSingleSpace',
 		'smartCharactersWithDoubleSpace'
 	],
-	#category : #'NECompletion-Model'
+	#category : #'NECompletion-Preferences'
 }
 
 { #category : #private }
 NECPreferences class >> availableEngines [
-	^ CompletionEngine withAllSubclasses
+	^ AbstractCompletionEngine allSubclasses
```

Comment:
Use subclasses of `AbstractCompletionEngine`, to cut the dependency.
Now the Morphic package depends on the core package, but not the other way around.